### PR TITLE
chore: Update folder localizations - WPB-14438

### PR DESCRIPTION
**Pull request by samwyndham**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-14438" title="WPB-14438" target="_blank">TaskWPB-14438</a>  [iOS] Fix folder related English localizations
  </td></table>
  <br />
 

### Issue

This PR updates a couple of localizations:

- The accessibly label for closing the folder picker during the move to folder flow
- The link text displayed in the the empty state of the folder picker used when filtering by folder

### Checklist

- [x] Title contains a reference JIRA issue number like `[WPB-XXX]`.
- [x] Description is filled and free of optional paragraphs.
- [ ] Adds/updates automated tests.

### CODE REVIEW

File: WireUI/Sources/WireFolderPickerUI/Resources/Localizable.xcstrings [hunk 1]
```diff
@@ -150,7 +150,7 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "How to add a conversation to a folder"
+            "value" : "Learn more"
```
Comment (caldrian) [inline WireUI/Sources/WireFolderPickerUI/Resources/Localizable.xcstrings:135]: I think this can't be correct.

**Review by johnxnguyen — APPROVED** [review]

**Review by agisilaos — APPROVED** [review]

File: WireUI/Sources/WireFolderPickerUI/Resources/Localizable.xcstrings (hunk 1 above)
Comment (johnxnguyen) [inline WireUI/Sources/WireFolderPickerUI/Resources/Localizable.xcstrings:135]: Hmm I see that all untranslated entries still have the old string, do we need to update those too or are new strings not used and the fallback is the base string?

Comment (samwyndham) [reply WireUI/Sources/WireFolderPickerUI/Resources/Localizable.xcstrings:135]: @johnxnguyen Oh right. That sucks. I'll take a look but perhaps I need to do this manually.

Comment (samwyndham) [reply WireUI/Sources/WireFolderPickerUI/Resources/Localizable.xcstrings:135]: I'll tag you into the discussion

Comment (samwyndham) [reply WireUI/Sources/WireFolderPickerUI/Resources/Localizable.xcstrings:135]: @johnxnguyen I've removed the translations as this seems to be the original state we have before sending localization to Crowdin. However, lets check what happens after crowdin does it's thing

**Pull request merged**